packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -569,13 +549,10 @@ const suggestedConfigurationContext = ref<{
 	modelConfiguration: null
 });
 const isFetching = ref(false);
-const modelFromDocumentHandler = ref();
-const modelFromDatasetHandler = ref();
+const modelFromDocumentTaskId = ref('');
+const modelFromDatasetTaskId = ref('');
 const isLoading = computed(
-	() =>
-		modelFromDocumentHandler.value?.isRunning ||
-		modelFromDatasetHandler.value?.isRunning ||
-		isFetching.value
+	() => modelFromDocumentTaskId.value || modelFromDatasetTaskId.value || isFetching.value
```

Comment:
What are the purposes of `modelFromDocumentTaskId` and `modelFromDatasetTaskId`? And why are they used in a boolean predicate when they are declared as strings?

Also these should really be model-configs, not models

Reply:
I used those to track the task currently running to show the loading spinner. yeah they should be model-configs I'll rename.

Reply:
Done